nautobot/dcim/models/device_component_templates.py

```diff
@@ -258,11 +259,16 @@ class Meta:
         unique_together = ("device_type", "name")
 
     def instantiate(self, device):
+        try:
+            status = Status.objects.get_for_model(Interface).get(slug=InterfaceStatusChoices.STATUS_ACTIVE)
+        except Status.DoesNotExist:
+            status = Status.objects.get_for_model(Interface).first()
```

Comment:
I know we're moving away from `cacheops` - I wonder if we should cache this in memory somewhere? When instantiating a DeviceType with a couple hundred InterfaceTemplates, is this going to hit the database each time?

Reply:
Do you recommend that I use an alternative memory cache, such as `django.core.cache.backends.locmem.LocMemCache`?
Or i add a TODO for this

Reply:
I wasn't thinking anything so fancy - I was thinking more of an approach like we have for `Prefix.STATUS_CONTAINER` where we just look it up and store it on the class for future re-use.